### PR TITLE
Dark mode styling

### PR DESCRIPTION
**Pull request by cdolfi**
This changes to the design overall, things to note: 
- dark mode 
- text and visualization color scheme 
- button changes 
- text styling 
- Layout orientation

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -100,178 +101,37 @@ def _project_list_query():
 import app_callbacks
 
 # CREATE APP OBJECT
-load_figure_template(["sandstone", "minty"])
+load_figure_template(["sandstone", "minty", "slate"])
+
+# stylesheet with the .dbc class
+dbc_css = "https://cdn.jsdelivr.net/gh/AnnMarieW/dash-bootstrap-templates/dbc.min.css"
+
```
Comment (JamesKunstle) [inline app.py:108]: it'd be nice to have some comments here about this, e.g. "creating a custom theme for site called 'custom_dark' in the plt_io.templates dictionary"

File: app.py [hunk 2]
```diff
@@ -100,178 +101,37 @@ def _project_list_query():
...
+plt_io.templates["custom_dark"] = plt_io.templates["slate"]
+plt_io.templates["custom_dark"]["layout"]["colorway"] = [
+    "#f8dd70",
+    "#c0bc5d",
+    "#8e9b4c",
+    "#62793d",
+    "#3c582d",
+    "#1c381d",
```
Comment (JamesKunstle) [inline app.py:116]: an in-line comment here of which colors these are (like 'dark-green', 'light-green', 'sandy') would be nice so we easily update them w/o looking up the color codes.

File: app.py [hunk 3]
```diff
@@ -100,178 +101,37 @@ def _project_list_query():
...
+]
+plt_io.templates.default = "custom_dark"
+
 app = dash.Dash(
     __name__,
     use_pages=True,
-    external_stylesheets=[dbc.themes.SANDSTONE],
+    external_stylesheets=[dbc.themes.SLATE, dbc_css],
```
Comment (JamesKunstle) [inline app.py:123]: comment why you have these style options in order, if order matters. e.g. "the base color pallet for the app is SLATE and is extended/overridden in some places by our custom dbc_css"

File: app.py [hunk 4]
```diff
@@ -100,178 +101,37 @@ def _project_list_query():
...
-                    width={"offset": 10},
-                )
-            ],
-        ),
-    ],
-    fluid=True,
-    className="dbc",
-    style={"padding-top": "1em"},
-)
+app.layout = layout
```
Comment (JamesKunstle) [inline app.py:134]: cool

File: app_callbacks.py [hunk 5]
```diff
@@ -187,7 +187,7 @@ def run_queries(repos):
         # jobs are either all ready
         if all([j.successful() for j in jobs]):
             logging.info([j.status for j in jobs])
-            return "Data Ready", "success"
+            return "Data Ready", "#B5B682"
```
Comment (JamesKunstle) [inline app_callbacks.py:190]: understood, also should have an in-line comment of what this color looks like for debugging / readability.

File: assets/color.css [hunk 6]
```diff
@@ -0,0 +1,6 @@
+h2, h3, h4, h5, h6 {
+    color: #8D9D92 !important;
```
Comment (JamesKunstle) [inline assets/color.css:2]: the '!important' is to override another CSS styling choice right?

File: pages/utils/graph_utils.py [hunk 7]
```diff
@@ -1,7 +1,7 @@
 import datetime as dt
 
 # list of graph color hex
-color_seq = ["#f8dd70", "#c0bc5d", "#8e9b4c", "#62793d", "#3c582d", "#1c381d"]
+color_seq = ["#B5B682", "#c0bc5d", "#6C8975", "#485B4E", "#3c582d", "#376D39"]
```
Comment (JamesKunstle) [inline pages/utils/graph_utils.py:4]: same ask of listing what these colors look like so we can edit them on-the-fly later.

File: pages/visualizations/overview/active_drifting_contributors.py [hunk 8]
```diff
@@ -162,10 +149,10 @@ def toggle_popover_4(n, is_open):
 def active_drifting_contributors_graph(repolist, interval, drift_interval, away_interval):
 
     if drift_interval is None or away_interval is None:
-        return dash.no_update, dash.no_update, dash.no_update
+        return dash.no_update, dash.no_update
```
Comment (JamesKunstle) [inline pages/visualizations/overview/active_drifting_contributors.py:152]: oop missed that

**Review by JamesKunstle — CHANGES_REQUESTED** [review]
just asking for some quality-of-life comments next to color codes so we can switch them out more quickly in the future. also, the loading animations are still blue, I don't know if we care about that.

File: app.py (hunk 3 above)
Comment (cdolfi) [reply app.py:123]: tbh on this not sure if the order matters here, the dbc_css a complement to the themes. https://github.com/AnnMarieW/dash-bootstrap-templates see the section on this file

File: assets/color.css (hunk 6 above)
Comment (cdolfi) [reply assets/color.css:2]: yes

**Pull request merged**